TROUBLESHOOTING.md

```diff
@@ -101,16 +101,32 @@ bind 'set horizontal-scroll-mode off'
 Terminal multiplexer tools like GNU `screen` or `tmux` are sometimes reported to fix the issues, though at other times they are reported as the _cause_ of the problem. They have their own configuration files to take into account.
 
 Further reference:
-* https://stackoverflow.com/questions/1133031/shell-prompt-line-wrapping-issue
-* https://superuser.com/questions/46948/any-way-to-fix-screens-mishandling-of-line-wrap-maybe-only-terminal-app
-* https://unix.stackexchange.com/questions/105958/terminal-prompt-not-wrapping-correctly
-* https://unix.stackexchange.com/questions/529377/terminal-long-line-wrapping
-* https://github.com/microsoft/WSL/issues/1436
+
+- https://stackoverflow.com/questions/1133031/shell-prompt-line-wrapping-issue
+- https://superuser.com/questions/46948/any-way-to-fix-screens-mishandling-of-line-wrap-maybe-only-terminal-app
+- https://unix.stackexchange.com/questions/105958/terminal-prompt-not-wrapping-correctly
+- https://unix.stackexchange.com/questions/529377/terminal-long-line-wrapping
+- https://github.com/microsoft/WSL/issues/1436
 
 If nothing seems to help, consider also using a different client-side terminal application:
-* Linux: xterm, KDE Konsole, GNOME Terminal
-* Mac: Terminal, iTerm2
-* Windows: PowerShell, PuTTY, WSL (Windows Subsystem for Linux)
+
+- Linux: xterm, KDE Konsole, GNOME Terminal
+- Mac: Terminal, iTerm2
+- Windows: PowerShell, PuTTY, WSL (Windows Subsystem for Linux)
+
+## "balena ssh" ends with permission denied when using RSA keys
+
+RSA keys generated with this command from github documentation: `ssh-keygen -t rsa -b 4096 -C "your_email@example.com"`
+
+It failed with this error:
+`Offering public key: /Users/<username>/.ssh/id_rsa RSA`
+`send_pubkey_test: no mutual signature algorithm`
```

Comment:
I gather that GitHub's documentation no longer advises `-t rsa`:
https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent

For this reason, I suggest not mentioning GitHub here:
```suggestion
Some operating systems or SSH packages have started disabling the RSA key type on security grounds. As a result, `balena ssh -v` may fail with error:
`Offering public key: /Users/<username>/.ssh/id_rsa RSA`
`send_pubkey_test: no mutual signature algorithm`
```